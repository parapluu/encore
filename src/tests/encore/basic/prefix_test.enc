class Main
    def main() : void
        let
            nothing = "blingbling"
            ping = true
            anders = true
        in{
            print(nothing);
            print(not(ping));
            print(not     ping);
            print(not anders);
            print(ping and anders);
         }
        