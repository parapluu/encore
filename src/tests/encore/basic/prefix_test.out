blingbling
bool<0>
bool<0>
bool<0>
bool<1>
